.github/workflows/Packages - EdFi.Ods.Populated.Template.TPDM.PostgreSQL.yml

```diff
@@ -0,0 +1,113 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: Packages - EdFi.Ods.Populated.Template.TPDM.PostgreSQL
+
+on:
+  push:
+    branches:
+      - main
+
+  workflow_dispatch:
+  repository_dispatch:
+    types: [triggered-from-implementation-repo]
```

Comment:
This dispatch type only gets triggered from the Implementation repo, we should also trigger it from the next repositories since TeamCity also monitors them:
- Ed-Fi-Standard
- Ed-Fi-TPDM-Artifacts

Reply:
added trigger for both repos